ntbs-service/Pages/Alerts/TransferRequestAction.cshtml.cs

```diff
@@ -25,6 +25,7 @@ public class TransferRequestActionModel : NotificationModelBase
         private readonly IAlertRepository _alertRepository;
         private readonly IAlertService _alertService;
         private readonly IReferenceDataRepository _referenceDataRepository;
+        private readonly IItemRepository<TreatmentEvent> _treatmentEventRepository;
         public ValidationService ValidationService;
         [BindProperty]

```

Comment:
Might be nice to get some whitespace in here to break things up a little bit... not explicitly in this changeset I know